civiwiki/api/models/article.py

```diff
@@ -2,6 +2,15 @@
 from django.db import models
 
 # Create your models here.
+class ArticleManager(models.Manager):
```

Comment:
Can we change this to **TopicManager**? also we should rename the file to **topic.py** now too.